compose/compose.yaml

```diff
@@ -0,0 +1,23 @@
+services:
+
+  redis-master:
+    container_name: redis-master
+    image: redis:latest
+    ports:
+      - "6379"
+
+  redis-replica:
+    container_name: redis-replica
+    image: redis:latest
+    ports:
+      - "6379"
+    command: redis-server --replicaof redis-master 6379
+    depends_on:
+      - redis-master
+
+  frontend:
+    container_name: frontend
+    image: cdrage/frontend
```

Comment:
Should be removed

Reply:
Yup, it is intended to be removed and hosted somewhere (quay.io).